app/controllers/course/assessment/assessments_controller.rb

```diff
@@ -22,6 +22,8 @@ class Course::Assessment::AssessmentsController < Course::Assessment::Controller
   def index
     @assessments = @assessments.ordered_by_date_and_title.with_submissions_by(current_user)
 
+    load_assessment_submission_counts if current_course_user&.staff?
```

Comment:
Double-check will this load if it's an systemAdmin / instanceAdmin which isn't a courseUser?

Also what if it's a systemAdmin / instanceAdmin that is a courseUser who is a student?